site/pages/dashboard.page.tsx

```diff
@@ -217,6 +236,9 @@ export const getServerSideProps = async (): Promise<{ props: DashboardProps }> =
                     disruptionEndDate: disruption.disruptionEndDate,
```

Comment:
This is where we should be taking the disruption.validity array, and if we have a disruption repeat, we override the last date with the date a year from now.

Reply:
Updated the code to run the end date check for repeating disruptions at server side